test/cmd/newapp.sh

```diff
@@ -18,6 +18,8 @@ os::cmd::expect_failure 'oc new-app unknownhubimage -o yaml'
 os::cmd::expect_failure_and_text 'oc new-app docker.io/node~https://github.com/openshift/nodejs-ex' 'the image match \"docker.io/node\" for source repository \"https://github.com/openshift/nodejs-ex\" does not appear to be a source-to-image builder.'
 os::cmd::expect_failure_and_text 'oc new-app https://github.com/openshift/rails-ex' 'the image match \"ruby\" for source repository \"https://github.com/openshift/rails-ex\" does not appear to be a source-to-image builder.'
 os::cmd::expect_success 'oc new-app https://github.com/openshift/rails-ex --strategy=source --dry-run'
+os::cmd::expect_success 'oc new-app  openshift/postgresql-92-centos7 --env-file=/tmp/envfile'
```

Comment:
you can fulfill my item (2) by doing:
`oc new-app someargs --env-file somefile -o yaml  | grep check_for_env_file_presence`